### PR TITLE
feat: add page footer.

### PR DESCRIPTION
**Pull request by jimsynz**
Also snuck in some code gardening because that's just how I is.

### CODE REVIEW

File: lib/playdate_web/live/user_settings_live.ex [hunk 1]
```diff
@@ -185,4 +186,11 @@ defmodule PlaydateWeb.UserSettingsLive do
         {:noreply, assign(socket, :marketing_form, form)}
     end
   end
+
+  defp button_class,
+    do:
+      "w-full md:w-40 py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-alembic-blue-dark-400 hover:bg-alembic-blue-dark-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-alembic-blue-pale-500"
+
+  defp mebbe(true, truthy, _), do: truthy
```
Comment (mbuhot) [inline lib/playdate_web/live/user_settings_live.ex:194]: is this private function used?

File: lib/playdate_web/templates/layout/app.html.sface [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="flex flex-col h-screen justify-between">
```
Comment (mbuhot) [inline lib/playdate_web/templates/layout/app.html.sface:1]: 👍 great if we can surface-all-the-things

File: test/playdate/games/planning_poker/change_current_issue_url_test.exs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 defmodule Playdate.Games.PlanningPoker.ChangeCurrentIssueUrlTest do
+  @moduledoc false
```
Comment (mbuhot) [inline test/playdate/games/planning_poker/change_current_issue_url_test.exs:2]: Feels a bit pedantic to opt-out of moduledocs in test scripts. Is doctor complaining about them?

File: lib/playdate_web/live/user_settings_live.ex (hunk 1 above)
Comment (jimsynz) [reply lib/playdate_web/live/user_settings_live.ex:194]: yeah, I just moved it down the file.

File: test/playdate/games/planning_poker/change_current_issue_url_test.exs (hunk 3 above)
Comment (jimsynz) [reply test/playdate/games/planning_poker/change_current_issue_url_test.exs:2]: credo complains about it in vscode for me, so I find it easier to just add them.

**Review by mbuhot — APPROVED** [review]
👍

**Pull request merged**